packages/nulp_elite/src/pages/search/ContentList.js

```diff
@@ -74,51 +81,89 @@ const ContentList = (props) => {
 
     const url = `http://localhost:3000/content/${URLSConfig.URLS.CONTENT.SEARCH}?orgdetails=orgName,email`;
     try {
-      const response = await contentService.getAllContents(
-        url,
-        data,
-        headers
-      );
+      const response = await contentService.getAllContents(url, data, headers);
       console.log(response.data.result);
       setData(response.data.result);
     } catch (error) {
-      console.log("error---",error);
-
+      console.log("error---", error);
       setError(error.message);
     } finally {
       console.log("finally---");
       setIsLoading(false);
     }
   };
 
-
   return (
     <div>
-      <Header/>
-      <Box sx={{background:'#2D2D2D',padding:'20px'}}>
-        <p style={{fontSize:'20px',fontWeight:'700',color:'#fff',paddingBottom:'5px',margin:'0'}}>Explore content related to your domain. Learn from well curated courses and content.</p>
-        <p style={{fontSize:'16px',fontWeight:'700',color:'#C1C1C1',margin:'0',paddingBottom:'30px'}}>Learn from well curated courses and content.</p>
+      <Header />
+      <Box sx={{ background: "#2D2D2D", padding: "20px" }}>
+        <p
+          style={{
+            fontSize: "20px",
+            fontWeight: "700",
+            color: "#fff",
+            paddingBottom: "5px",
+            margin: "0",
+          }}
+        >
+          Explore content related to your domain. Learn from well curated
+          courses and content.
+        </p>
+        <p
+          style={{
+            fontSize: "16px",
+            fontWeight: "700",
+            color: "#C1C1C1",
+            margin: "0",
+            paddingBottom: "30px",
+          }}
+        >
+          Learn from well curated courses and content.
+        </p>
         <Search></Search>
       </Box>
 
       <Container maxWidth="xxl" role="main" className="container-pb">
-        <Box style={{margin:'20px 0'}}>
-          <Filter/>
+        <Box style={{ margin: "20px 0" }}>
+          <Filter
```

Comment:
@mahajanmahesh935 , as per our dicussion, please pass here Framework API response values, we can not hardcode it as these values might change later.